### ISSUE TITLE
App doesn't run

### ISSUE THREAD

**Issue by danastasio** [chmouel/gnome-next-meeting-applet#27]
* Gnome Next Meeting applet version: latest flatpak version
* Python version: 3.10.2
* Operating System: Fedora Workstation 35

### Description

Launching the app from the app grid in GNOME doesn't do anything at all. Launching from CLI gives errors.

### What I Did

Command
```flatpak run com.chmouel.gnomeNextMeetingApplet -v```

Output
```
DEBUG:root:loading configfile: /home/$USER/.var/app/com.chmouel.gnomeNextMeetingApplet/config/gnome-next-meeting-applet/config.yaml
DEBUG:root:Watching calendar Birthdays & Anniversaries
DEBUG:root:Watching calendar calendar
DEBUG:root:Watching calendar Personal
DEBUG:root:Watching calendar Work Calendar
Traceback (most recent call last):
  File "/app/lib/python3.9/site-packages/gnma/gnome_online_account_cal.py", line 130, in view_objects_added
    self.handle_new_or_modified_objects(view,
  File "/app/lib/python3.9/site-packages/gnma/gnome_online_account_cal.py", line 260, in handle_new_or_modified_objects
    ical_time_modified = mod_prop.get_lastmodified()
AttributeError: 'NoneType' object has no attribute 'get_lastmodified'
Traceback (most recent call last):
  File "/app/lib/python3.9/site-packages/gnma/gnome_online_account_cal.py", line 187, in recurrence_generated
    ical_time_modified = mod_prop.get_lastmodified()
AttributeError: 'NoneType' object has no attribute 'get_lastmodified'
```

Thanks!

**Comment by chmouel** [comment]
Hey thanks for the report again, any chance you can tell me what kind of calendar sources you have (i.e: google, exchange or others?) ?

I guess you don't have problems when running Gnome Calendar and can see all your events?

I am sending a tentative fix/workaround, are you able to test the fix locally ? (ie: with checking the pull request and pip3 install --user)

**Comment by danastasio** [comment]
If I checkout the repo and install using pip, I get the following error:

```
Traceback (most recent call last):
  File "/home/$USER/Projects/gnome-next-meeting-applet/bin/gnome-next-meeting-applet", line 5, in <module>
    from gnma.cli import cli
  File "/home/$USER/Projects/gnome-next-meeting-applet/lib64/python3.10/site-packages/gnma/cli.py", line 5, in <module>
    from gnma import applet, dbusclient
  File "/home/$USER/Projects/gnome-next-meeting-applet/lib64/python3.10/site-packages/gnma/applet.py", line 11, in <module>
    gi.require_version("AppIndicator3", "0.1")
  File "/home/$USER/Projects/gnome-next-meeting-applet/lib64/python3.10/site-packages/gi/__init__.py", line 126, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace AppIndicator3 not available
```

If I checkout the repo and build using flatpak-builder, I do not get any errors, but nothing appears in the notification tray either.

I have a few different calendar providers. Exchange (OWS), GMail, and a self-hosted caldav provider as well. You are correct, if I open gnome calendar, all events appear as expected.

**Comment by chmouel** [comment]
humm maybe some of those events don't have the same expected format, i only have access to gmail accounts currently.. 

let me do a release and ping you so it would be easier for you to update, make take a few days tho.

**Comment by danastasio** [comment]
No worries! Thanks for all the help. If I can get you more logs or any more data just let me know!

**Comment by Rombusevil** [comment]
> If I checkout the repo and install using pip, I get the following error:
> 
> ```
> Traceback (most recent call last):
>   File "/home/$USER/Projects/gnome-next-meeting-applet/bin/gnome-next-meeting-applet", line 5, in <module>
>     from gnma.cli import cli
>   File "/home/$USER/Projects/gnome-next-meeting-applet/lib64/python3.10/site-packages/gnma/cli.py", line 5, in <module>
>     from gnma import applet, dbusclient
>   File "/home/$USER/Projects/gnome-next-meeting-applet/lib64/python3.10/site-packages/gnma/applet.py", line 11, in <module>
>     gi.require_version("AppIndicator3", "0.1")
>   File "/home/$USER/Projects/gnome-next-meeting-applet/lib64/python3.10/site-packages/gi/__init__.py", line 126, in require_version
>     raise ValueError('Namespace %s not available' % namespace)
> ValueError: Namespace AppIndicator3 not available
> ```
> 
> If I checkout the repo and build using flatpak-builder, I do not get any errors, but nothing appears in the notification tray either.
> 
> I have a few different calendar providers. Exchange (OWS), GMail, and a self-hosted caldav provider as well. You are correct, if I open gnome calendar, all events appear as expected.

Oh this is one of the things that should be working on the PR I made with the ayatana dep. Maybe fedora behaves like debian in that respect @danastasio @chmouel

**Comment by chmouel** [comment]
I have merged @Rombusevil  PR in 2.8.0, let me know if that works for you @danastasio
